tests/BenchmarkDotNet.IntegrationTests.Classic/Exporters/ApprovalTest.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
+using ApprovalTests;
+using ApprovalTests.Namers;
+using ApprovalTests.Reporters;
+using BenchmarkDotNet.Columns;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Exporters;
+using BenchmarkDotNet.Exporters.Csv;
+using BenchmarkDotNet.Exporters.Json;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Tests.Mocks;
+using Xunit;
+
+namespace BenchmarkDotNet.IntegrationTests.Classic.Exporters
+{
+    [UseReporter(typeof(XUnit2Reporter))]
+    public class ApprovalTest
```

Comment:
For now, `BenchmarkDotNet.IntegrationTests.Classic` is a good place for `ApprovalTests` because the corresponded NuGet package doesn't support .NET Core. In the future (with dotnet-preview3+), we will be able to move it to `BenchmarkDotNet.Tests` without pain. I suggest to put a comment here which is explain why do we keep the `ApprovalTest` in `IntegrationTests` and contains a `TODO` for the future.